gui/wxpython/core/toolboxes.py

```diff
@@ -208,7 +208,7 @@ def getMenudataFile(userRootFile, newFile, fallback):
                 fh.write(xml)
                 fh.close()
                 return menudataFile
-            except:
+            except Exception:
```

Comment:
Good reason to have just Exception there. This may develop in the future, either splitting the try-except or catching some while letting others create a traceback.